src/cli/bin.js

```diff
@@ -15,12 +15,6 @@ updateNotifier({
 }).notify()
 
 const cli = yargs
-  .option('q', {
-    alias: 'quiet',
-    desc: 'suppress output',
-    type: 'boolean',
-    coerce: (quiet) => { if (quiet) { utils.disablePrinting() } }
-  })
```

Comment:
This removes this option for **all** commands. We need to find a way of removing it for just the ipfs.files.add command.